packages/dotcom-ui-shell/src/components/Shell.tsx

```diff
@@ -36,25 +42,15 @@ function Shell(props: TShellProps) {
     // should be found by the browser's speculative parser.
     ...props.scripts,
     ...props.resourceHints,
-    // TODO: abstract font URLs into 'core branding' package
-    'https://www.ft.com/__origami/service/build/v2/files/o-fonts-assets@1.3.2/MetricWeb-Regular.woff',
-    'https://www.ft.com/__origami/service/build/v2/files/o-fonts-assets@1.3.2/MetricWeb-Semibold.woff',
-    'https://www.ft.com/__origami/service/build/v2/files/o-fonts-assets@1.3.2/FinancierDisplayWeb-Regular.woff',
-    'https://www.ft.com/__origami/service/build/v2/files/o-fonts-assets@1.3.2/FinancierDisplayWeb-Bold.woff'
+    ...fontFaceURLs
   ]
 
   return (
     <html
       {...formatAttributeNames(props.htmlAttributes)}
       lang="en-GB"
-      className="no-js core"
-      style={{
-        // TODO: abstract styles into 'core branding' package
-        // Enable use of 100vw which does not account for the scroll bar
-        overflowX: 'hidden',
-        backgroundColor: '#fff1e5',
-        color: '#33302e'
-      }}>
+      className={`no-js core ${loadCustomFontsClassNames}`}
```

Comment:
Having access to the `<html>` element (and not relying on `.n-layout`) is so much nicer!